### DISCUSSION TITLE
Self test error

### DISCUSSION THREAD

**Discussion by Arve1996** [sanni/cartreader#861] [Need help]
I recently built the HW 5 revision and whenever I use the self test I get a short between pins error with D35 and D36 displayed. I used my multimeter and there is no short in continuity mode, but I get a little bit of resistance before showing no short. I have tried with and without the six cartridge slot PCB with the same results.

**Comment by sanni** [comment]
Check the GB slot's D1 and D2 pins. Could be a tiny solder blob there.

**Comment by Arve1996** [comment]
Hi there, thank you for you quick reply! I decided to re-do the main pcb since I suspected something went wrong at some point and some of the pins might have a short smh. That fixed the short between pins and ground short that sometimes appeared; however, when I try to dump any save file (So far GBA and N64 Games) it is in blank despite any progress is in the real cartridge, but when I try to dump the ROM, it does work! I tried to backup Pokemon Stadium 2 and Pokemon Ruby, to be more specific. Also, I used the N64 controller test and this works as a charm. so I don't know what would cause the issue. I'm using the latest release (beta), btw.

**Reply by Arve1996** [reply]
Quick update! I tried with a GBC Game and I always get the "HEADER CHECKSUM ERROR".
